src/examples/Makefile.in

```diff
@@ -9,7 +9,8 @@ top_srcdir=@top_srcdir@
 
 CC=@CC@
 CPPFLAGS=-I$(top_srcdir)/src @CPPFLAGS@ \
-	-I$(top_srcdir)/src/json-lib -I$(top_srcdir)/src/utils -I$(top_srcdir)/src/modules
+	-I$(top_srcdir)/src/json-lib -I$(top_srcdir)/src/utils -I$(top_srcdir)/src/modules \
+	-I/usr/include/amd64
```

Comment:
Any include directories from outside the source-tree should be specified via autoconf.